_sections/act_and_prepare/en/02-do/07-stock-up-on-food-and-essentials.md

```diff
@@ -23,5 +23,13 @@ Thread:
 "Experts aren't telling you to stock up on essentials because they think you're going to run out of food & society is going to collapse. It's
 because a [few days of panic buying & high demands cause more panic & stress systems unnecessarily](https://twitter.com/firefoxx66/status/1233666678841597952). Also, being in line with hundreds of people isn't wise. Go off peak. Buy a little more than what you need. Preparedness isn't about doing nothing, but also about not overreacting. It's about doing your part to put the [\*slack in our systems\*](https://twitter.com/firefoxx66/status/1233666678841597952) so that short disruptions are smooth as possible, can absorb extra load, & resources remain available for those most in need.
 
-Don't bother buying bottled water; tap water is very unlikely to be impacted and anyway, boiling would be adequate if it turned out it were
-needed for unforeseen reasons. Intestinal symptoms of coronavirus are very rare; there's no reason to expect that supplies of toilet paper will dry up. Get what you need. Leave the rest for others.
+Buy food and other supplies responsibly. Some tips:
+ - Go to the store at off-peak hours. 
+ - Make a list so you can be efficient.
+ - Retain a 6ft distance between yourself and others at all times. 
```

Comment:
this is the recommendation in the US. Do we know what the recommendation is in other countries? Should we also include the number in meters?